frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/k8s-resource.ts

```diff
@@ -5,6 +5,18 @@ import { K8sResourceCommon, Patch } from '../../extensions/console-types';
 import { consoleFetchJSON as coFetchJSON } from '../fetch';
 import { selectorToString, resourceURL } from './k8s-utils';
 
+const abbrBlacklist = ['ASS'];
+
+/**
+ * It creates an abbreviation for given kind.
+ * @param kind Kind for which the abbreviation is being generated.
+ * @return Abbreviation for given kind.
+ * * */
+export const kindToAbbr = (kind) => {
```

Comment:
@fhlavac based on @christianvogt's comments, can we add a `todo` here to investigate removing this fallback in the SDK? We'll need to verify that API discovery **always** sets the abbreviation and this is never exercised.

Otherwise, LGTM 👍

Reply:
https://github.com/openshift/console/blob/6ba4a8c42e65b77a0600e89b80c437af25c8793b/frontend/public/module/k8s/get-resources.ts#L197

We generate the abbreviation for all models and then we also merge in abbreviations coming from metadata.
The point is that we should only be doing this once in a central location for API discovery.

Reply:
@christianvogt ahh, gotcha. This PR is just to unblock [HAC-197](https://issues.redhat.com/browse/HAC-197) but if we want to do this refactoring now, I can take a look. Maybe it's better to do it now since we may not need this function to be migrated then...

Reply:
missed your point for the first time

Reply:
Ah. Thanks for clarifying @christianvogt - I misunderstood. @fhlavac based on your screenshot above, we should have a look at why some resources (IE `Binding` and `Endpoints`) don't have models/abbr loaded. They're possibly coming from another API and perhaps we need to move the code Christian linked to that more centralized place - perhaps an action or reducer?

Reply:
I'm very curious as to why those models are missing abbreviations.